integration/gethnetwork/geth_binaries.go

```diff
@@ -43,7 +43,8 @@ func EnsureBinariesExist(version string) (string, error) {
 	cmd := exec.Command(shCmd, gethScript, fmt.Sprintf("%s=%s", versionFlag, version))
 	cmd.Stderr = os.Stderr
 
-	if _, err := cmd.Output(); err != nil {
+	if out, err := cmd.Output(); err != nil {
+		fmt.Printf("%s\n", out)
```

Comment:
Is this left in on purpose?

Reply:
I think it's useful to print that output in case something goes wrong